### PR TITLE
refactor: update nuxt guide

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

resolves LOG-9381

- support `title` for code snippets like Docusaurus
- update Nuxt guide according to the new template

## Testing

image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/assets/docs/fragments/_sign_in_explanation.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+1. Your app invokes the sign-in method.
+2. The user is redirected to the Logto sign-in page. For native apps, the system browser is opened.
+3. The user signs in and is redirected back to your app (configured as the redirect URI).
+
+Regarding redirect-based sign-in:
+
+1. This authentication process follows the [OpenID Connect (OIDC)](https://openid.net/specs/openid-connect-core-1_0.html) protocol, and Logto enforces the Authorization Code Flow with PKCE to ensure the security of the authentication process.
+2. If you have multiple apps, you can use the same authentication server (Logto), and the user only needs to sign in once.
+
+To learn more about the rationale and benefits of redirect-based sign-in, see [Logto sign-in experience explained](https://docs.logto.io/docs/tutorials/get-started/sign-in-experience).
```
Comment (gao-sun) [inline packages/console/src/assets/docs/fragments/_sign_in_explanation.md:18]: this doc is wip

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/assets/docs/fragments/_sign_in_explanation.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Before we dive into the details, here's a quick overview of the end-user experience. The sign-in process can be simplified as follows:
+
+```mermaid
+graph LR
+    A(Your app) -->|1. Invoke sign-in| B(Logto)
+    B -->|2. Finish sign-in| A
+```
+
+1. Your app invokes the sign-in method.
+2. The user is redirected to the Logto sign-in page. For native apps, the system browser is opened.
```
Comment (simeng-li) [inline packages/console/src/assets/docs/fragments/_sign_in_explanation.md:10]: System browser sounds tricky, e.g. iOS popup webview is not the same as opening a Safari browser?

File: packages/console/src/assets/docs/fragments/_sign_in_explanation.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+3. The user signs in and is redirected back to your app (configured as the redirect URI).
+
+Regarding redirect-based sign-in:
+
+1. This authentication process follows the [OpenID Connect (OIDC)](https://openid.net/specs/openid-connect-core-1_0.html) protocol, and Logto enforces the Authorization Code Flow with PKCE to ensure the security of the authentication process.
```
Comment (simeng-li) [inline packages/console/src/assets/docs/fragments/_sign_in_explanation.md:15]: PKCE is not enforced for traditional apps, should we make it configurable?  Looks like all types of applications may share this fragment.  e.g. Nuxt is a traditional web app framework and PKCE is not enforced.

File: packages/console/src/assets/docs/fragments/_sign_in_explanation.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+2. If you have multiple apps, you can use the same authentication server (Logto), and the user only needs to sign in once.
```
Comment (simeng-li) [inline packages/console/src/assets/docs/fragments/_sign_in_explanation.md:16]: How about the same identity provider or identity management system?  Feels like sharing the same identity and auth session is the fundamental part of a universal sign-in experience.

Comment (xiaoyijun) [reply packages/console/src/assets/docs/fragments/_sign_in_explanation.md:16]: Of course! Here’s the translation:

"The user only needs to sign in once." This phrase might cause misunderstandings. Users might think they can use an access token from one app in another app. It should be phrased as "Once the user has logged in to one app, Logto will automatically complete the sign in process for them when they initiate the sign in process in other apps."

File: packages/console/src/assets/docs/fragments/_sign_in_explanation.md (hunk 2 above)
Comment (gao-sun) [reply packages/console/src/assets/docs/fragments/_sign_in_explanation.md:10]: it's actually the system browser. iOS only allows the system browser to be used.

Comment (gao-sun) [reply packages/console/src/assets/docs/fragments/_sign_in_explanation.md:10]: all webviews, safari controllers, even the Chrome app in iOS use the same browser kernel

File: packages/console/src/assets/docs/fragments/_sign_in_explanation.md (hunk 3 above)
Comment (gao-sun) [reply packages/console/src/assets/docs/fragments/_sign_in_explanation.md:15]: right. let me update

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by xiaoyijun — APPROVED** [review]

**Pull request merged**